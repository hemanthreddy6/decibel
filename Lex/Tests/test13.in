clip: audio = audio(SIN,1.0,440);
clip = clip + audio(COS,1.0,440);
clip = clip[5:15];
play clip^10;
