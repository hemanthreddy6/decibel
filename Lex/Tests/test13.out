AUDIO
IDENTIFIER
=
AUDIO
(
SIN
,
FLOAT
,
NUMBER
)
;
IDENTIFIER
=
IDENTIFIER
+
AUDIO
(
COS
,
FLOAT
,
NUMBER
)
;
IDENTIFIER
=
IDENTIFIER
[
NUMBER
:
NUMBER
]
;
PLAY
IDENTIFIER
^
NUMBER
;
